### PR TITLE
[FEATURE] Frontend

### PR DESCRIPTION
**Pull request by andrii-shchur**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                data: JSON.stringify({ title: title, text: text }),
+                contentType: "application/json",
+                success: function(response) {
+                    const result = response.detection;
+                    const verdict = result > 0.5 ? "Fake" : "False"
+                    $("#result").html(`<h4>Verdict: <span class="badge badge-primary">${verdict}</span></h4>`);
+                },
+            });
+
+            $("#result").html(`<h4>Verdict: <span class="badge badge-primary">Amidst the digital storm, the detector contemplates its decision with epic solemnity...</span></h4>`);
```
Comment (BogdanaGonserovska) [inline index.html:50]: I think `verdict` isn't an appropriate word, change to 'result' maybe or smth like that, or just omit this and have only this `${verdict}` value

**Review by BogdanaGonserovska — APPROVED** [review]
Great!

File: index.html (hunk 1 above)
Comment (andrii-shchur) [reply index.html:50]: i wanted it to sound epic and cool😣

Comment (BogdanaGonserovska) [reply index.html:50]: I'm sorry, then you can leave it as it is. I'm very sorry I've made you feel bad(

Comment (andrii-shchur) [reply index.html:50]: i fixed it already so don't worry ƪ(˘⌣˘)ʃ

Comment (BogdanaGonserovska) [reply index.html:50]: Oh nooooooooo

**Pull request merged**